main.tf

```diff
@@ -333,7 +334,7 @@ su -c "tabcmd --accepteula" - tableau_srv
 echo "#TSMCMD - initial user"
 tabcmd initialuser --server 'localhost:80' --username "$TAB_ADMIN_USER" --password "$TAB_ADMIN_PASSWORD"
 
-# DQdashboard Customise
+#echo "#DQDashboard Customise"
```

Comment:
I thought you want the echo

Reply:
Yes, I do!
Thx for spotting the comment char - it is a typo.